types/joi/index.d.ts

```diff
@@ -146,7 +146,14 @@ export interface ValidationErrorItem {
     type: string;
     path: string;
     options?: ValidationOptions;
-    context?: any;
+    context?: {
+        [key: string]: any;
+        value?: any;
```

Comment:
Why do you add `value?` when there's already `[key: string]: any;` defined that allows any property?